code/modules/projectiles/guns/throw/pielauncher.dm

```diff
@@ -17,7 +17,7 @@
 /obj/item/gun/throw/piecannon/Initialize(mapload)
 	. = ..()
 	for(var/i in 1 to max_capacity)
-		var/obj/item/reagent_containers/food/snacks/pie/P = new /obj/item/reagent_containers/food/snacks/pie(src)
+		var/obj/item/food/snacks/pie/P = new /obj/item/food/snacks/pie(src)
```

Comment:
same question as above, can this be simplified to `var/obj/item/food/snacks/pie/P = new(src)`?